mne_bids/tests/test_write.py

```diff
@@ -1075,3 +1076,44 @@ def test_write_anat(_bids_validate):
         anat_dir = write_anat(bids_root, subject_id, t1w_mgh, session_id,
                               acq, deface=True, landmarks=fail_landmarks,
                               verbose=True, overwrite=True)
+
+
+def test_write_raw_pathlike():
```

Comment:
how much is this code a copy-paste from other tests? how much did your tests increase testing running time?

couldn't you just update an existing test?

Reply:
With those new tests disabled:

```
============================================== slowest 20 test durations ==============================================
12.16s call     mne_bids/tests/test_write.py::test_fif
8.54s call     mne_bids/tests/test_write.py::test_write_anat
7.98s call     mne_bids/tests/test_write.py::test_edf
6.90s call     mne_bids/tests/test_write.py::test_kit
6.00s call     mne_bids/tests/test_write.py::test_set
5.78s call     mne_bids/tests/test_write.py::test_vhdr
5.22s call     mne_bids/tests/test_write.py::test_ctf
4.57s call     mne_bids/tests/test_write.py::test_bti
3.81s call     mne_bids/tests/test_write.py::test_bdf
0.09s call     mne_bids/tests/test_write.py::test_get_anonymization_daysback

pytest tests/test_write.py  84.20s user 11.53s system 152% cpu 1:02.88 total
```

With the tests enabled:

```
============================================== slowest 20 test durations ==============================================
9.97s call     mne_bids/tests/test_write.py::test_fif
8.56s call     mne_bids/tests/test_write.py::test_write_anat
8.14s call     mne_bids/tests/test_write.py::test_edf
6.51s call     mne_bids/tests/test_write.py::test_kit
6.27s call     mne_bids/tests/test_write.py::test_set
6.11s call     mne_bids/tests/test_write.py::test_vhdr
5.01s call     mne_bids/tests/test_write.py::test_ctf
4.56s call     mne_bids/tests/test_write.py::test_bti
4.05s call     mne_bids/tests/test_write.py::test_bdf
0.24s call     mne_bids/tests/test_write.py::test_write_raw_pathlike
0.12s call     mne_bids/tests/test_write.py::test_write_anat_pathlike
0.08s call     mne_bids/tests/test_write.py::test_get_anonymization_daysback

pytest tests/test_write.py  85.71s user 11.01s system 159% cpu 1:00.76 total
```

So they these tests take about half a second on my MacBook Pro (obscenely fast SSD)

Reply:
Can still move those tests into existing tests if you prefer

Reply:
ok it's fast enough